cloudbees-rollout/README.md

```diff
@@ -0,0 +1,3 @@
+# CloudBees Rollout Examples
```

Comment:
Why is there a readme.md file for rollout (old product name)? You have a readme.md file for feature flags.

Reply:
Removed